lib/post.php

```diff
@@ -140,6 +140,11 @@ public function post_content() {
 
 		if ( function_exists( 'wpmarkdown_html_to_markdown' ) ) {
 			$content = wpmarkdown_html_to_markdown( $content );
+		} else if(class_exists("WPCom_Markdown")) {
```

Comment:
Per the [WordPress style guide](https://make.wordpress.org/core/handbook/coding-standards/php/#space-usage), I believe we'll want to add some whitespace around the parenthesis before we merge.